guides/common/modules/proc_registering-hosts-by-using-the-bootstrap-script.adoc

```diff
@@ -1,9 +1,12 @@
 [id="Registering_Hosts_by_Using_the_Bootstrap_Script_{context}"]
 = Registering hosts by using the bootstrap script
 
-**Deprecated** Use xref:Registering_Hosts_by_Using_Global_Registration_{context}[] instead.
+You can use the bootstrap script to automate content registration and Puppet configuration.
+
+:FeatureName: bootstrap script
+:use-instead: xref:Registering_Hosts_by_Using_Global_Registration_{context}[]
```

Comment:
```suggestion
:feature-name: bootstrap script
:feature-alternative: xref:Registering_Hosts_by_Using_Global_Registration_{context}[]
```
Alternative: "alternative-url" instead of "feature-alternative". Pascal case and kebab case together is confusing to me.